ludwig/benchmarking/summary_dataclasses.py

```diff
@@ -138,3 +325,60 @@ def build_experiments_diff(
         experimental_summary=experimental_summary,
         metrics=metrics,
     )
+
+
+def build_resource_usage_summary(path):
```

Comment:
nit: missing type hints. Would also be good to add a docstring with the detail that this is returning the ResourceUsageSummary with the mean across all runs, if there are multiple runs.